rtos/source/TARGET_CORTEX/mbed_boot.c

```diff
@@ -64,8 +64,10 @@
 #include "mbed_error.h"
 #include "mbed_mpu_mgmt.h"
 
+
 int main(void);
 static void mbed_cpy_nvic(void);
+void mbed_rtos_init_singleton_mutex(void);
```

Comment:
This can be properly publicly declared in mbed_boot.h. mbed_rtx_rtos.c doesn't seem to have its own header, but it includes mbed_boot.h, and that's where `mbed_rtos_start` is declared.

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/14bde89d6f58208b501ebd0ee57da21fd266d09b..5d94fd4d7361306d4655ee30ae9ac3c507d32bd1) everything is already internal documentation.